docker/test/docker-labels-arm64.golden.json

```diff
@@ -0,0 +1,20 @@
+{
+  "io.k8s.description": "Workflow engine for microservice orchestration",
+  "io.openshift.min-cpu": "1",
+  "io.openshift.min-memory": "512Mi",
+  "io.openshift.non-scalable": "false",
+  "io.openshift.tags": "bpmn,orchestration,workflow",
+  "org.opencontainers.image.authors": "zeebe@camunda.com",
+  "org.opencontainers.image.base.digest": "sha256:ce46be0c4b4edd9f519e99ad68a6b5765abe577fbf1662d8ad2550838eb29823",
```

Comment:
:wrench: If we wanted to share the golden file, we could do it using jq substitutions (much like we do with $DATE, $REVISION, etc.). I think the Identity team did the same in order to support renovate automating updates to the pinned SHA.

See:

- https://github.com/camunda-cloud/identity/blob/17cdf68a9f204a0afa02377591b8d9638f34e593/docker/verify.sh#L44
- https://github.com/camunda-cloud/identity/blob/17cdf68a9f204a0afa02377591b8d9638f34e593/docker/docker-labels.golden.json#L9

Reply:
That way, we may get away with a single golden file.

Reply:
done with https://github.com/camunda/zeebe/pull/11003/commits/0c26e0f9e80195604230dffed6a0bcc4a8435b2f